src/react-medium-image-zoom.js

```diff
@@ -138,6 +140,7 @@ export default class ImageZoom extends Component {
       delete this.portalInstance
       delete this.portal
     }
+    this.props.onUnzoom()
```

Comment:
`componentDidUpdate` is the culprit here (the `else if` needs 1 more condition). I think I have a fix but need to step out for a few.  BRB -- I might do a separate PR to fix that up, merge that, then merge this. Other than that, looks good!